tests/org.jboss.tools.mylyn.ui.bot.test/pom.xml

```diff
@@ -96,6 +96,109 @@
                 <systemProperties>-Dawt.toolkit=sun.lwawt.macosx.LWCToolkit</systemProperties>
             </properties>
         </profile>
+		<profile>
+			<id>cdi11-tests</id>
+			<properties>
+				<suiteClass>org.jboss.tools.mylyn.bot.test.MylynSuiteTest</suiteClass>
```

Comment:
Do you really need this property variable? I can't find it used anywhere. If you do need it, I guess, that "cdi11-tests" profile id is not what you want ;-)